### PR TITLE
deltachat-desktop: 1.36.3 -> 1.36.4

### PR DESCRIPTION
**Pull request by dotlambda**
###### Description of changes
Diff: https://github.com/deltachat/deltachat-desktop/compare/v1.36.3...v1.36.4

Changelog: https://github.com/deltachat/deltachat-desktop/blob/v1.36.4/CHANGELOG.md

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

cc @Simon-Laux

### CODE REVIEW

File: pkgs/applications/networking/instant-messengers/deltachat-desktop/default.nix [hunk 1]
```diff
@@ -21,12 +21,12 @@
 
 let
   libdeltachat' = libdeltachat.overrideAttrs (old: rec {
-    version = "1.112.7";
+    version = "1.112.8";
     src = fetchFromGitHub {
       owner = "deltachat";
       repo = "deltachat-core-rust";
       rev = version;
```
Comment (lilyinstarlight) [inline pkgs/applications/networking/instant-messengers/deltachat-desktop/default.nix:28]: Did this actually build at the time this was merged?

It looks like upstream revs here are `v${version}` https://github.com/deltachat/deltachat-core-rust/releases/tag/v1.112.8 and this fetch 404's locally for me (noticed while nixpkgs-review'ing another PR)

Comment (lilyinstarlight) [reply pkgs/applications/networking/instant-messengers/deltachat-desktop/default.nix:28]: (Let me know if I'm just doing a dumb. Here was the output from the build in nixpkgs-review:)

```
trying https://github.com/deltachat/deltachat-core-rust/archive/1.112.8.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0    14    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
curl: (22) The requested URL returned error: 404
error: cannot download source from any mirror
```

Comment (lilyinstarlight) [reply pkgs/applications/networking/instant-messengers/deltachat-desktop/default.nix:28]: Okay, I don't think I'm just holding it wrong. Hydra is failing too for deltachat-desktop: https://hydra.nixos.org/build/217089656

Comment (dotlambda) [reply pkgs/applications/networking/instant-messengers/deltachat-desktop/default.nix:28]: Oops, my bad. I just copied the hash from the expression for libdeltachat.

**Pull request merged**